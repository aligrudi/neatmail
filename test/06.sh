# neatmail test
./mail ex $1 <<EOF
:1,2mv /dev/null
:w
EOF
cat $1
