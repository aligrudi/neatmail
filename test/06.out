From root Tue Dec 21 01:01:01 2015
From: A <a@a.eu>
To: B <b@b.eu>
Subject: Message from A
Message-Id: <msg001>
Date: Tue, 21 Dec 2015 01:01:00 +0100

This is A.

